<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-JLSUG9BU" failures="2" tests="5" name="Test" time="21.434" errors="0" timestamp="2025-08-30T18:00:42 IST">
  <testcase classname="com.testcase.DemoKartTestSuite" name="tc001_addToCartWithoutLogin" time="0.162"/>
  <testcase classname="com.testcase.DemoKartTestSuite" name="tc002_loginWithValidCredentials" time="0.297"/>
  <testcase classname="com.testcase.DemoKartTestSuite" name="tc003_welcomeBackToastOnProductPage" time="1.116"/>
  <testcase classname="com.testcase.DemoKartTestSuite" name="tc004_addToCartAndValidateUrl" time="0.453">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [Phone 15 Pro Max has been added to your cart] but found [iPhone 15 Pro Max has been added to your cart]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [Phone 15 Pro Max has been added to your cart] but found [iPhone 15 Pro Max has been added to your cart]
at com.testcase.DemoKartTestSuite.tc004_addToCartAndValidateUrl(DemoKartTestSuite.java:69)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- tc004_addToCartAndValidateUrl -->
  <testcase classname="com.testcase.DemoKartTestSuite" name="tc005_checkoutAndOrderConfirmation" time="4.284">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [Order placed successfully!] but found [Payment failed]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [Order placed successfully!] but found [Payment failed]
at com.testcase.DemoKartTestSuite.tc005_checkoutAndOrderConfirmation(DemoKartTestSuite.java:80)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- tc005_checkoutAndOrderConfirmation -->
</testsuite> <!-- Test -->
